floris/utilities.py

```diff
@@ -230,11 +232,8 @@ def rotate_coordinates_rel_west(
         wind_directions (NDArrayFloat): Series of wind directions to base the rotation
         coordinates (NDArrayFloat): Series of coordinates to rotate with shape (N coordinates, 3)
             so that each element of the array coordinates[i] yields a three-component coordinate
```

Comment:
Need to update the docstring to reflect new input arguments and a different output type

Reply:
Updated. Not sure how to best post the updates here?